docs/changelogs/CHANGELOG-2.22.md

```diff
@@ -0,0 +1,481 @@
+# Kubermatic 2.22
+
+- [v2.22.0](#v2220)
+
+## [v2.22.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.22.0)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.22/tutorials-howtos/upgrading/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure usercluster reconciliations will not cause resource exhaustion on seed clusters.
```

Comment:
Not sure if it should be `usercluster`, `user cluster`, `user cluster's` or `user-cluster`...
```suggestion
Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.22/tutorials-howtos/upgrading/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure user cluster reconciliations will not cause resource exhaustion on seed clusters.
```

Reply:
"user cluster" is the best out of them I think.